File: src/main/java/com/yogpc/qp/CreativeTabQuarryPlus.java
package com.yogpc.qp;

import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

public class CreativeTabQuarryPlus extends CreativeTabs {

    public CreativeTabQuarryPlus() {
        super(QuarryPlus.Mod_Name);
    }

    @Override
    public ItemStack getTabIconItem() {
        return new ItemStack(QuarryPlusI.blockQuarry);
    }
}

File: src/main/java/com/yogpc/qp/CreativeTabQuarryPlus.scala
package com.yogpc.qp

import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack

class CreativeTabQuarryPlus extends CreativeTabs(QuarryPlus.Mod_Name) {

    override def getTabIconItem = new ItemStack(QuarryPlusI.blockQuarry)

}